CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-### [4.1.9](https://github.com/wagoid/commitlint-github-action/compare/v4.1.8...v4.1.9) (2021-10-11)
+### [4.1.10](https://github.com/wagoid/commitlint-github-action/compare/v4.1.9...v4.1.10) (2022-01-28)
```

Comment:
Could you rollback changes to this file? It's automatically updated once I merge to master